dts/arm/alif/balletto_rtss_common.dtsi

```diff
@@ -286,6 +286,11 @@
 			interrupts = <124 0>;
 			reg-shift = <2>;
 			current-speed = <115200>;
+			/* refer datasheet for
```

Comment:
Please remove the comments. Those don't bring any information and clutters the files.